test/integration/setup.js

```diff
@@ -24,15 +24,15 @@ const request = require('request');
 before(function(done) {
 	const that = this;
 
-	that.baseUrl = 'http://localhost:' + config.port + '/';
-	that.app = null;
-	that.last = {};
-	that.navigate = createNavigator(that.baseUrl, that.last);
+	this.baseUrl = 'http://localhost:' + config.port + '/';
```

Comment:
* why is `that` now `this`? 
* if `that` is now `this`, why does line 25 still exist?

Reply:
That's a mouthful!

We didn't need to use the `that` reference in most of the function. However within the callback on line `25` we do need to reference the `this` from the parent, which is why we have to keep `that`.